database/migrations/2024_07_24_104411_create_books_table.php

```diff
@@ -13,8 +13,8 @@ public function up(): void
     {
         Schema::create('books', function (Blueprint $table) {
             $table->id();
-            $table->string('book');
-            $table->string('author_name');
+            $table->string('book_name');
+            $table->string('author');
```

Comment:
- book_name isinmlendirmesi pratik değil. Sebebi ise zaten books tablosundaki kolonlar aksi belirtilmedikçe book ile alakalı datalardır. nasıl id nin başına extra book_id diye yazılmasına gerek yoksa zaten kitapların id si olduğu anlaşılııyorsa name için de aynısı geçerli. 

İstisna durumları author_name gibi bilgiler tutacağın zaman olur.

(Değişiklik yapmana gerek yok sadece bil yeter)